examples/ssao/main.js

```diff
@@ -0,0 +1,701 @@
+( function () {
+    'use strict';
+
+    var ExampleOSGJS = window.ExampleOSGJS;
+
+    var OSG = window.OSG;
+    var osg = OSG.osg;
+    var osgViewer = OSG.osgViewer;
+    var osgShader = OSG.osgShader;
+    var osgUtil = OSG.osgUtil;
+    var Texture = osg.Texture;
+
+    var $ = window.$;
+    var P = window.P;
+
+    // Generates MAX_MIP_LEVEL depth mipmaps.
+    // Do not forget to change the code in ssaoFragment.glsl
+    // because the mipmaps are passed manually using texture units
+    var MAX_MIP_LEVEL = 5;
+
+    var shaderProcessor = new osgShader.ShaderProcessor();
+
+    // Loads a glTF model and update
+    // the current displayed scene in the Example
+    var loadFiles = function ( example, files, sceneName ) {
+
+        var promise = OSG.osgDB.Registry.instance().getReaderWriterForExtension( 'gltf' )
```

Comment:
For now, readNodeURL uses:
` var extension = url.substr( url.lastIndexOf( '.' ) + 1 );`
But the glTF Reader/Writer can handle files. We should either add a new method, or modify this one (but modifying it without changing its name does not make sense)